docs/source/technical_design/utilities/scripts/openKAT_install.sh

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/bash
+
+# Installation for Debian 11
+# echo (https://docs.openkat.nl/technical_design/debianinstall.html)
+
+# echo "number of arguments '$#'"
+
+# Checking if at least 1 parameter was given
+if [ $# -eq 0 ] || [ $# -gt 3 ]; then
+	echo "Usage: ./openKAT_install.sh [debian version (11 or 12)] [openKAT version] [no_super_user]"
```

Comment:
I think many of us share that preference because it's quite common in Linux-like environments to have lowercased apps/scripts/commands and usually also separated by `-` (dashes).